src/Message.java

```diff
@@ -0,0 +1,21 @@
+public class Message {
+
+    Players player;
+    int actionType;
+
+    public Message (Players playerNumber, int actionType) {
+        this.player = playerNumber;
+        this.actionType = actionType;
+    }
```

Comment:
The `Message` should look like this right?

https://github.com/TimonRey/PingPongApp/blob/89ee8306d258e9b731add35f5c784c8258fe3749/src/main/java/PingPong.java#L48-L51

So what you get from a `Message` is
* `buttonId`: it's a string
* `actionType`: also a string

Reply:
No, the __button__ variable is a *string* but the __actionType__ is an *int*...just a number